testkit/tests/system_api.rs

```diff
@@ -52,3 +53,12 @@ fn test_network() {
     assert_eq!(info.protocol_version, PROTOCOL_MAJOR_VERSION);
     assert!(info.services.is_empty());
 }
+
+#[test]
+fn test_consensus_status() {
```

Comment:
I suppose, one validator isn't the right test case here. And you need to check for `false` values too.

Reply:
It will be changed during ECR-1665